src/encoder.rs

```diff
@@ -487,6 +488,19 @@ pub struct FrameInvariants<T: Pixel> {
   /// should be ignored. Invalid frames occur when a subgop is prematurely
   /// ended, for example, by a key frame or the end of the video.
   pub invalid: bool,
+  /// Motion vectors to the _original_ reference frames (not reconstructed).
+  /// Used for lookahead purposes.
+  pub mvs: Vec<FrameMotionVectors>,
```

Comment:
This variable name is confusing, even with the comments. Consider we have ReferenceFrame::frame_mvs and FrameInvariants::mvs. lookahead_mvs or something would be better, even if more verbose.